### PR TITLE
Staking rename -> validator manager

### LINKED ISSUE #545: Rename staking -> validatormanager
**Context and scope**
Rename staking folders to validatormanager, since poa validators don't have staking.

### PR DESCRIPTION
**Pull request by iansuvak**
## Why this should be merged
Closes #545

## How this was tested

based off of staking-rewards so once those tests are fixed these should pass as well 

## How is this documented

### CODE REVIEW

File: tests/flows/staking/poa_to_pos.go [hunk 1]
```diff
@@ -6,8 +6,8 @@ import (
 
 	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/subnet-evm/accounts/abi/bind"
-	nativetokenstakingmanager "github.com/ava-labs/teleporter/abi-bindings/go/staking/NativeTokenStakingManager"
```
Comment (minghinmatthewlam) [inline tests/flows/staking/poa_to_pos.go:9]: I think similar to the contracts, this test folder should be renamed to `validatormanager`, same with the pkg name

File: tests/utils/staking.go [hunk 2]
```diff
@@ -21,10 +21,10 @@ import (
 	"github.com/ava-labs/subnet-evm/precompile/contracts/warp"
 	predicateutils "github.com/ava-labs/subnet-evm/predicate"
 	exampleerc20 "github.com/ava-labs/teleporter/abi-bindings/go/mocks/ExampleERC20"
-	erc20tokenstakingmanager "github.com/ava-labs/teleporter/abi-bindings/go/staking/ERC20TokenStakingManager"
-	examplerewardcalculator "github.com/ava-labs/teleporter/abi-bindings/go/staking/ExampleRewardCalculator"
-	nativetokenstakingmanager "github.com/ava-labs/teleporter/abi-bindings/go/staking/NativeTokenStakingManager"
-	poavalidatormanager "github.com/ava-labs/teleporter/abi-bindings/go/staking/PoAValidatorManager"
+	erc20tokenstakingmanager "github.com/ava-labs/teleporter/abi-bindings/go/validatorManager/ERC20TokenStakingManager"
```
Comment (minghinmatthewlam) [inline tests/utils/staking.go:24]: this utils can also be renamed to validator manager

**Comment by michaelkaplan13** [comment]
Honestly not sure if we have a standard here or not yet...but what do people prefer between `validator-manager` and `validatorManager` for the directory convention?

**Comment by michaelkaplan13** [comment]
> Honestly not sure if we have a standard here or not yet...but what do people prefer between `validator-manager` and `validatorManager` for the directory convention?

`signature-aggregator` was the only example I could find that has me leaning lower case hyphenated right now

https://github.com/ava-labs/awm-relayer/tree/main/signature-aggregator

**Comment by iansuvak** [comment]
> > Honestly not sure if we have a standard here or not yet...but what do people prefer between `validator-manager` and `validatorManager` for the directory convention?
> 
> `signature-aggregator` was the only example I could find that has me leaning lower case hyphenated right now
> 
> https://github.com/ava-labs/awm-relayer/tree/main/signature-aggregator

I agree I will make the change

**Review by bernard-avalabs — APPROVED** [review]

**Review by geoff-vball — APPROVED** [review]
👍

**Pull request merged**